websites/M/MangaHere/presence.ts

```diff
@@ -0,0 +1,144 @@
+const presence = new Presence({
+  clientId: "831262912815300638"
+}),
+  browsingStamp = Math.floor(Date.now() / 1000);
+
+// let username: string;
+
+presence.on("UpdateData", async () => {
+  const data: PresenceData = {
+    largeImageKey: "mangahere"
+  },
+    pathname = document.location.pathname,
+    hostname = document.location.hostname,
+    ganres = ["martial-arts", "action", "school-life",
+              "sci-fi", "yoi", "shotacon", "mystery", "shoujo",
+              "ecchi", "doujinshi", "lolicon", "adventure", "romance",
+              "gender-bender", "harem", "sports", "webtoons", "comedy",
+              "shounen-ai", "josei", "shoujo-ai", "adult", "fantasy", "supernatural",
+              "psychological", "yuri", "one-shot", "historical", "drama",
+              "seinen", "mature", "smut", "horror", "shounen", "slice-of-life",
+              "tragedy", "mecha"
+            ];
+
+  if (hostname === "mangahere.cc" || hostname === "www.mangahere.cc") {
```

Comment:
That's the same thing in different variations lol, just include one

Reply:
I did it just in case xD I'll fix it now.